apps/feature-first-hasura-console-poc/src/app/Console.tsx

```diff
@@ -0,0 +1,29 @@
+import { useConsoleInfoStore } from '@feature-first-hasura-console-poc/hasura-features';
+import { Neon } from './features/Neon';
+import { Prometheus } from './features/Prometheus';
+
+export function Console() {
+  const state = useConsoleInfoStore((state) => state);
+
+  return (
+    <>
+      <br />
```

Comment:
**joke:** Were you cold @NoriSte? (BrrrBrrrrBrrr)

Reply:
😂😂😂